### ISSUE TITLE
Cannot create multiple invoices with the same label

### ISSUE THREAD

**Issue by luke-jr** [ElementsProject/lightning#5684]
```
{
   "code": 900,
   "message": "Duplicate label '...'"
}
```

This error doesn't make sense. Labels aren't supposed to be unique...

**Comment by d4amenace** [comment]
Hello,

From the readme for invoice:

"The label must be a unique string or number (which is treated as a string, so “01” is different from “1”); it is never revealed to other nodes on the lightning network, but it can be used to query the status of this invoice."

**Comment by kristapsk** [comment]
"label" is this context is unique invoice id, maybe not the best naming, but it is what it is.

**Comment by cdecker** [comment]
Yeah, naming is hard. The purpose is to give the caller a unique way to refer to an invoice, even if the connection gets dropped during the call (weak form of idempotency).
